src/Symfony/Component/Console/Descriptor/ApplicationDescription.php

```diff
@@ -144,9 +144,11 @@ private function sortCommands(array $commands)
         }
         ksort($namespacedCommands);
 
-        foreach ($namespacedCommands as &$commands) {
-            ksort($commands);
+        foreach ($namespacedCommands as &$commandsSet) {
+            ksort($commandsSet);
         }
+        // unset reference to keep scope clear
+        unset($commandsSet);
```

Comment:
this is not necessary. It will be removed anyway when reaching the end of the method on next line